exercises/practice/two-fer/src/test/kotlin/TwoFerTest.kt

```diff
@@ -10,21 +10,18 @@ class TwoFerTest {
     }
 
     @Test
-    @Ignore
```

Comment:
This should be ignored as in practice exericses, all tests are ignored except for the first.